### DISCUSSION TITLE
datalad get not available error

### DISCUSSION THREAD

**Discussion by alecristia** [LAAC-LSCP/ChildProject#288] [Q&A]
I created a YODA dataset and pushed it to github following [these instructions](http://docs.datalad.org/en/stable/generated/man/datalad-create-sibling-github.html) and now I cannot get the file:

$ datalad get draft.Rmd 
[ERROR  ] not available; (Note that these git remotes have annex-ignore set: origin) [get(/Users/acristia/Documents/gitrepos/aclew10kq1/draft.Rmd)]                                
get(error): draft.Rmd (file) [not available; (Note that these git remotes have annex-ignore set: origin)]

I also cannot view it online: https://github.com/aclew/aclew10kq1/blob/master/draft.Rmd

Should I specify files like that to be not annexed?
https://github.com/aclew/aclew10kq1/blob/master/.gitattributes

And also, what of the files that are in this supposed annex -- how can I get them?

Thanks a lot in advance!

**Comment by alecristia** [comment]
for good measure, I tried adding all md's files to gitattributes ignore, but I still cannot get the file.

My local version of the README looks fine, but the online one is just a pointer.

Sorry -- this must be a super basic error I made, some step I skipped in my instructions...

**Comment by lucasgautheron** [comment]
Hello,

GitHub does not support annexd files -- so you will need an additional sibling/remote from which these files can be fetched if you absolutely need to store these files in the annex.
Do you have to use git-annex for this repository? (i.e. does your dataset include large files that cannot be directly versioned in git ?)
If the large files are all inside of subdatasets, this should not be an issue (you could just version everything in the superdataset and leave it on GitHub!)

**Reply by alecristia** [reply]
no, no large files here! so I don't need annex at all.

Hmm where did I go wrong, do you think, when creating the folder such that git-annex was 'activated'? If you are not sure, no worries.

**Reply by lucasgautheron** [reply]
did you create it with `datalad create` ?
in any case,  I think a `.gitattributes` file with the following rule should work:

```
* annex.largefiles=nothing
```

**Reply by alecristia** [reply]
ok, i think this probably fixed my problems for the future. I hope to remember to do things correctly next time! I think this is a great idea: https://github.com/LAAC-LSCP/ChildProject/issues/301
